daemon/daemon.go

```diff
@@ -213,6 +204,21 @@ func (daemon *Daemon) restore() error {
 					logrus.Errorf("Failed to restore %s with containerd: %s", c.ID, err)
 					return
 				}
+
+				// we call Mount and then Unmount to get BaseFs of the container
+				if err := daemon.Mount(c); err != nil {
```

Comment:
This will fail if it's already mounted though, right?
Can we check if it's already mounted before logging an error?
We have too many mount related warning messages in the daemon logs as is.